db/sg_replicate_cfg.go

```diff
@@ -163,6 +163,14 @@ func (rc *ReplicationConfig) ValidateReplication(fromConfig bool) (err error) {
 		}
 	}
 
+	if !rc.ConflictResolutionType.IsValid() {
+		return base.HTTPErrorf(http.StatusBadRequest, "Conflict resolution type is invalid")
```

Comment:
Include valid types in error message, similar to the `direction` validation. Should define in a constant for easier test assertion.